modules/configuring-network-subnets.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// networking/network-optimization.adoc
+
+[id='Configuring-network-subnets-{context}']
+= Configuring network subnets
```

Comment:
What does configuring the subnet help the user achieve? What are the benefits and why would you do this?